python/sanity_checks/sanity_check.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import ingestion.standardized_columns as std_col
+
+
+def generate_cols(df: pd.DataFrame):
```

Comment:
Do we use this file? It looks like you copied the code to the `airflow/dags/sanity_check.py` file and use that in the pipeline. We should use the code that we run our tests on.